packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -22,7 +22,7 @@ export const AdminUI = ({
     error = Error,
     ...props
 }: AdminUIProps) => (
-    <ScopedCssBaseline enableColorScheme>
+    <ScopedCssBaseline enableColorScheme sx={{ height: '100vh' }}>
```

Comment:
This will break things. Can't you do it in the Error component instead?

Reply:
We can do it, yes. But in this case, the admin will be bigger than the view height as we can see with the scrollbar: 
![image](https://github.com/marmelab/react-admin/assets/131013150/6e04c037-c1b1-437b-8622-c77c24ed36cc)